recipe/meta.yaml

```diff
@@ -12,9 +13,9 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: {{ build }}
+  skip: True  # [py<37]
```

Comment:
It should be `py<36`, see https://github.com/googleapis/google-auth-library-python-oauthlib/blob/830651eff490c47aeb3ba0b6b27afa9f92ddccdf/setup.py#L48